release/src/main/groovy/TestScripts.groovy

```diff
@@ -209,16 +220,17 @@ class TestScripts {
                </profile>
              </profiles>
         </settings>
-        """
-       def cmd = "mvn ${args} -s ${settings.absolutePath} -Ptestrel -B"
-       String path = System.getenv("PATH");
-       // Set the path on jenkins executors to use a recent maven
-       // MAVEN_HOME is not set on some executors, so default to 3.5.2
-       String maven_home = System.getenv("MAVEN_HOME") ?: '/home/jenkins/tools/maven/apache-maven-3.5.4'
-       println "Using maven ${maven_home}"
-       def mvnPath = "${maven_home}/bin"
-       def setPath = "export PATH=\"${mvnPath}:${path}\" && "
-       return _execute(setPath + cmd)
+         """
+     }
+     def cmd = "mvn ${args} -s ${settings.absolutePath} -Ptestrel -B"
```

Comment:
Shall we also add "-U" in addition to this change ?

Reply:
Sorry later on I found my initial investigation in https://github.com/apache/beam/issues/32824#issuecomment-2419679190 was wrong. In particular, the test workflow 8 tasks in parallel, each spin up an individual maven local repository, and delete after finish. So there isn't exist scenario that bad local repository cached can interfere the next run.

Using `-U` tag will cause the workflow making more requests to maven snapshot repository, which is known to be less stable in terms of availability, and I am afraid it would worsen rate limiting

Reply:
Ack. Thanks.